test/edu/csus/ecs/pc2/ui/team/CreateTeamsTSVTest.java

```diff
@@ -0,0 +1,73 @@
+package edu.csus.ecs.pc2.ui.team;
+
+import java.io.File;
+import java.util.List;
+
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+import edu.csus.ecs.pc2.core.util.AbstractTestCase;
+
+/**
+ * Unit tests.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu> F
+ */
+public class CreateTeamsTSVTest extends AbstractTestCase {
+
+    /**
+     * Test loadAccounts
+     * 
+     * @throws Exception
+     */
+    public void testloadAccounts() throws Exception {
+
+        String dataDir = getDataDirectory(this.getName());
+        String name = dataDir + File.separator + CreateTeamsTSV.TEAMS_JSON_FILENAME;
+
+        assertFileExists(name);
+//        ensureDirectory(dataDir);
```

Comment:
I was surprised to see this line commented out... wouldn't that be a valid, useful operation?

Reply:
It only needs to be run once to create the input data directory, so after the first time
it is just using extra cycles.

Reply:
But doesn't that mean that if down-the-road a previous test (contained the "first time") is removed for some reason (we've done that in the past), then this test suddenly could fail when it shouldn't?   It doesn't seem to me like avoiding the "extra cycles" is worth the long-view risk.  But whatever; at least I understand why you did it.